mgmtsystem_nonconformity/models/mgmtsystem_nonconformity_stage.py

```diff
@@ -2,17 +2,18 @@
 # Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models, fields, _
-
+from odoo import models, fields, _
 
 _STATES = [
     ('draft', _('Draft')),
-    ('analysis', _('Analysis')),
-    ('pending', _('Action Plan')),
+    ('analysis', _('Analysis & Planing')),
+    ('pending', _('Pending Approval')),
     ('open', _('In Progress')),
+    ('evaluation', _('In Evaluation')),
     ('done', _('Closed')),
     ('cancel', _('Cancelled')),
 ]
+_STATES_DICT = dict(_STATES)
 
 
 class MgmtsystemNonconformityStage(models.Model):
```

Comment:
IMO this should be switched to use [base_kanban_stage](https://github.com/OCA/server-tools/tree/10.0/base_kanban_stage), which would provide more features.